src/Apps/Settings/Routes/MyCollection/Components/MyCollectionArtworkGrid.tsx

```diff
@@ -1,22 +1,77 @@
+import { useDismissibleContext } from "@artsy/dismissible"
 import ArtworkGrid from "Components/ArtworkGrid/ArtworkGrid"
+import { FC } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import { Text } from "@artsy/palette"
+import { PROGRESSIVE_ONBOARDING } from "Components/ProgressiveOnboarding/progressiveOnboardingKeys"
+import { _FragmentRefs } from "relay-runtime"
 
-export const MyCollectionArtworkGrid = createFragmentContainer(ArtworkGrid, {
-  artworks: graphql`
-    fragment MyCollectionArtworkGrid_artworks on MyCollectionConnection {
-      edges {
-        node {
-          id
-          slug
-          href
-          internalID
-          image(includeAll: true) {
-            aspectRatio
+const POPOVER_KEY = PROGRESSIVE_ONBOARDING.startSelling
+
+interface MyCollectionArtworkGridProps {
+  artworks: _FragmentRefs<"MyCollectionArtworkGrid_artworks">
```

Comment:
What is `_FragmentRefs`? I have never used it before. And would `MyCollectionArtworkGrid_artworks$data` work here as well? 👀

Reply:
And in any case, we should (always) be using `useFragment` here vs our old school `createFragmentContainer`

Reply:
@olerichter00  I've never used it before either, but without it get this typescript message:

![image](https://github.com/artsy/force/assets/3934579/ab0a327e-5ff9-4947-a106-0c27e7780cca)

Reply:
Ah, it should work if you use [ArtworkGridContainer](https://github.com/artsy/force/blob/fdc195db92ebcad255819f5c611dac309de070e1/src/Components/ArtworkGrid/ArtworkGrid.tsx#L65) instead of `ArtworkGrid`. in line [25](https://github.com/artsy/force/pull/13812/files#diff-a57b784f4ea583c258b8362e8e383903c4b5decfd9da4f5793f9300ff31aceefR25) because `ArtworkGrid` doesn't like `MyCollectionConnection` so instead of wrapping the artwork grid fragment container, [MyCollectionArtworkGrid.tsx](https://github.com/artsy/force/pull/13812/files#diff-a57b784f4ea583c258b8362e8e383903c4b5decfd9da4f5793f9300ff31aceef) needs to wrap the artwork grid itself.

Reply:
oh, that works, thanks @olerichter00!